### DISCUSSION TITLE
Tabs are triggered with right mouse click

### DISCUSSION THREAD

**Discussion by badsyntax** [radix-ui/primitives#1112] [Help] [answered]
Hi there. 

I've noticed the tabs are triggered using the right mouse click. This behaviour can be observed here: https://www.radix-ui.com/docs/primitives/components/tabs (Chrome, MacOS)

It seems this behaviour is not intentional, see comment here: https://github.com/radix-ui/primitives/blob/2fbc3470cd997e5ea642ec8089c43f0b55dc6ef2/packages/react/tabs/src/Tabs.tsx#L177-L183

I believe the right-mouse-click is triggered via the focus event here: https://github.com/radix-ui/primitives/blob/2fbc3470cd997e5ea642ec8089c43f0b55dc6ef2/packages/react/tabs/src/Tabs.tsx#L187-L194

Possibly related tabs focus issue: https://github.com/radix-ui/primitives/issues/1047

Let me know if you'd prefer for me to create an issue for this.

**Comment by benoitgrelard** [comment, marked as answer]
Hey @badsyntax, nice seeing you here! 🙂
Yeah you're totally right about that one, I've converted into an issue and will address now.

Cheers!
